courses/models.py

```diff
@@ -220,6 +231,9 @@ class Program(TimestampedModel, PageProperties, ValidateOnSaveMixin):
     live = models.BooleanField(default=False)
     products = GenericRelation(Product, related_query_name="programs")
     is_external = models.BooleanField(default=False)
+    partner = models.ForeignKey(
```

Comment:
Ideally, We should have had a common model e.g. `Courseware` having all the common fields from `Course, Program` models e.g. `live, title, readable_id`. And the `Course, Program` models would inherit from that. Let me know what you think. Maybe we can keep this here for now and do a full refactor of all these in another PR?

Reply:
Yes, I think we should do this in a separate PR.